ansible/roles/eksa/tasks/main.yaml

```
@@ -7,82 +47,52 @@
-    src: "/home/{{ ansible_user }}/{{ cluster_name }}/generated/{{ cluster_name }}-eks-a-cluster.kubeconfig"
-    dest: "/home/{{ ansible_user }}/.kube/config"
-    remote_src: true
-    mode: '0600'
-
-- name: "Display cluster creation result"
-  ansible.builtin.debug:
-    var: create_cluster_result.stdout_lines
\ No newline at end of file
+    cmd: "eksctl anywhere create cluster -f {{ build_dir }}/cluster.yaml"
+  delegate_to: localhost
+  run_once: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)